ReleaseHistory.md

```diff
@@ -23,6 +23,7 @@
 - DEP: Upgrade `Microsoft.Security.Utilities` from 6.2.1 to 6.5.0. [#788](https://github.com/microsoft/sarif-pattern-matcher/pull/788)
 - BRK: Replaced `MatchLengthToDecode` property of `MatchExpression` with new `Base64EncodingMatch` class to support detecting Base64 string from strings of specific length range. [#790](https://github.com/microsoft/sarif-pattern-matcher/pull/790)
 - FNC: Update `SEC101/041.RabbitMqCredentials` in `Security` to check loose credential combinations. [#788](https://github.com/microsoft/sarif-pattern-matcher/pull/788)
+- FND: Remove the secret format assumption in static validation for rule `SEC101/044.NpmCredentials` in `Security`. [#799](https://github.com/microsoft/sarif-pattern-matcher/pull/799)
```

Comment:
>assumption in static validation [](http://example.com/codeflow?start=32&length=31)

Nit, mention assumption was secret contained upper, lower, and digit

Reply:
updated, check the new record